coresimd/mod.rs

```diff
@@ -126,7 +126,7 @@ pub mod arch {
     /// See the [module documentation](../index.html) for more details.
     #[cfg(any(target_arch = "powerpc", dox))]
     #[doc(cfg(target_arch = "powerpc"))]
-    #[unstable(feature = "stdsimd", issue = "0")]
+    #[unstable(feature = "stdsimd", issue = "27731")]
```

Comment:
For powerpc we don't have a tracking issue in the stdsimd repo but there is one issue open about altivec in the rust-lang/rust repo that we might want to recycle here: https://github.com/rust-lang/rust/issues/42743 cc @lu-zero I don't know if you wanted to open an issue in the stdsimd repo tracking powerpc status like we did for x86, arm, and msa, but if you do don't forget to cross-link that rust-lang/rust issue.

Reply:
I can open an issue about this as well. Do you have a specific format for the tracker ?

Reply:
No specific format, what I've done in the past is hack a script that just translate the vendor C API to rust types, and then formats it as a list for the tracker. Each vendor is different, so there is not much to reuse here :/